db/migrations/013_endpoint_validation.up.sql

```diff
@@ -82,7 +82,7 @@ BEGIN
     IF (NEW.spec).model.registry IS NULL OR trim((NEW.spec).model.registry) = ''
     THEN
         RAISE sqlstate 'PGRST'
-            USING message = '{"code": "10011","message": "spec.model.registry is required","hint": "Provide model registry"}',
+            USING message = '{"code": "10009","message": "spec.model.registry is required","hint": "Provide model registry"}',
```

Comment:
modifying the previous migration sql file is not a good behavior, maybe we can create a new migration sql file, drop the function and create a new one.